### PR TITLE
Add sparklines to graph recent occurrences of a problem

### PR DESCRIPTION
**Pull request by danielbeardsley**
Now we include a sparkline of the timestamps from recent notices.
Two graphs give at-a-glance access to near and far history.

image

### CODE REVIEW

File: app/views/notices/_summary.html.haml [hunk 1]
```diff
@@ -23,6 +23,13 @@
       %th Similar
       %td= problem.notices_count - 1
     %tr
+      - two_weeks = problem.timestamps_since(Time.now.to_i - 86400 * 14)
+      %th Graph (2w @ 12h)
+      %td= sparkline(two_weeks, Time.now.to_i - 86400*14, 28)
+    %tr
+      %th Graph (1d @ 1h)
```
Comment (marczych) [inline app/views/notices/_summary.html.haml]: I think that `Graph` should be something more descriptive like `Occurrences`, `Instances`, or `Activity`.

File: app/views/notices/_summary.html.haml [hunk 2]
```diff
@@ -23,6 +23,13 @@
       %th Similar
       %td= problem.notices_count - 1
     %tr
+      - two_weeks = problem.timestamps_since(Time.now.to_i - 86400 * 14)
```
Comment (marczych) [inline app/views/notices/_summary.html.haml]: Can't we pass in all of the timestamps because `sparkline()`filters values before the start time anyway?

Comment (danielbeardsley) [inline app/views/notices/_summary.html.haml]: We could cause they get filtered out, but we don't need to retrieve them *all* from the DB. It significantly slows things down if there are thousands. I thought they were already loaded at first, but now I'm pretty sure they aren't. The distribution of messages and user-agents seems to be stored in the problem itself, not built dynamically on every load.

Comment (marczych) [inline app/views/notices/_summary.html.haml]: Ahh, I didn't realize that `notices.where(...)` did a DB query.

**Comment by danielbeardsley** [comment]
We don't use this branch anymore (we use ifixit-production, which already has this branch merged in)

**Pull request closed without merging**